api/src/routes/protected/settings.ts

```diff
@@ -609,7 +653,7 @@ ${isLinkSentWithinLimitTTL}`
           type: 'success'
         } as const;
       } catch (err) {
-        fastify.log.error(err);
+        logger.error(err);
```

Comment:
Are we sure only logging the error will give us enough context?

Reply:
Well, it's hard to say because this code only executes if something unexpected happens.

Hopefully it's enough and, if not, it should give as a hint about what we can improve.